File: 241003/색종이의 총 넓이/the-total-area-of-colored-paper.py
N = int(input())
arr = [[0]*201 for i in range(201)]

for i in range(N):
    x, y = map(int, input().split())
    for m in range(x, x+8):
        for n in range(y, y+8):
            arr[m+100][n+100] = 1

cnt = 0
for i in range(201):
    for j in range(201):
        if arr[i][j] == 1:
            cnt += 1

print(cnt)

File: 250106/색종이의 총 넓이/the-total-area-of-colored-paper.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int arr[201][201] = {};
    int N;
    cin >> N;
    for (int i=0; i<N; i++) {
        int x, y;
        cin >> x >> y;
        for (int j=x; j<x+8; j++) {
            for (int k=y; k<y+8; k++) {
                arr[j+100][k+100] = 1;
            }
        }
    }
    int cnt = 0;
    for (int i=0; i<201; i++) {
        for (int j=0; j<201; j++) {
            if (arr[i][j] == 1) cnt++;
        }
    }
    cout << cnt;
    return 0;
}